cla_public/templates/accessibility-statement.html

```diff
@@ -64,19 +73,23 @@ <h2 class="govuk-heading-m">{{ _('Non-accessible content') }}</h2>
     {% trans %}The content listed below is non-accessible for the following reasons.{% endtrans %}
   </p>
 
-  <h3 class="govuk-heading-m">{{ _('Non compliance with the accessibility regulations') }}</h3>
+  <h3 class="govuk-heading-m">{{ _('Non-compliance with the accessibility regulations') }}</h3>
 
-  <ul class="govuk-list govuk-list--bullet">
-    <li>
-      {% trans %}Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable).{% endtrans %}
-    </li>
-    <li>
-      {% trans %}Some page elements are not available in Welsh. This fails WCAG 2.1 3.1.1 A (Language of Page).{% endtrans %}
-    </li>
-    <li>
-      {% trans %}When a user enters figures into form fields it is not clear that the field does not accept currency symbols. This fails WCAG 2.1 2.4.6 AA (Headings and Labels).{% endtrans %}
-    </li>
-  </ul>
+  <h4 class="govuk-heading-s">{{ _('Time-out') }}</h4>
+
+  <p class="govuk-body">
+    {% trans %}Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable).{% endtrans %}
+  </p>
+  <p class="govuk-body">
+    {% trans %}We are experiencing intermittent problems with the time-out feature where users are unexpectedly timed-out without the time-out warning appearing. We are currently investigating the source of this issue.{% endtrans %}
+  </p>
+  <h4 class="govuk-heading-s">{{ _('Hide this page') }}</h4>
+  <p class="govuk-body">
+    {% trans %}The ‘Hide this page’ button does not accurately describe the link destination. This fails WCAG 2.1 1.3.1 A (Info and relationships) and 2.4.6 AA (Headings and Labels).{% endtrans %}
+  </p>
+  <p class="govuk-body">
+    {% trans %}We understand that ‘Hide this page’ does not describe the link destination. We have purposely designed the feature to protect vulnerable users and provide a quick exit from the service rather than take users to a specific destination.{% endtrans %}
+  </p>
```

Comment:
I don't see any Welsh translation for these. Are these pending translation return?

Reply:
Pending the new batch of translations.